zero/common/gpt2.py

```diff
@@ -197,5 +197,6 @@ def gpt2_builder():
 
     CONFIG['dataset'] = os.environ['DATA']
     CONFIG['tokenizer'] = os.environ['TOKENIZER']
+    CONFIG['model']['numel'] = get_model_size(build_model())

```

Comment:
This will cause model to be built twice?

Reply:
Is it because of zero init context?

Reply:
Yes, we can also compute model size locally and then do all-reduce. But the size may have bias due to padding.

Reply:
ok

Reply:
I can get model size in zero init context.